script/packages/linux

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#/ Usage: script/packages/linux [VERSION]
+#/
+#/ WARNING: You should not need to call this directly. Please create packages using
+#/ `script/package [platform]` or `bundle exec rake package[platform]`
+#/
+#/ Builds a linux distributable package for licensed for a given and licensed VERSION.
+#/ Packages are of the form licensed-$VERSION-linux-x64.tar.gz and contain a `./licensed` executable
+#/ Built packages are placed in the <root>/pkg directory.
+#/
+#/ If calling from a non-linux OS, docker is used to build a linux binary
+#/
+#/ OPTIONS:
+#/   [VERSION]         (optional, default to current git branch or SHA1) version of licensed to build exe at
+#/
+
+set -e
```

Comment:
FYI if you want to go a little further with Bash safety, you could use `set -euo pipefail`, which I've been putting into new scripts.  See https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail